.env.example

```
@@ -6,10 +6,26 @@
 # REQUIRED CONFIGURATION
 # ============================================
 
+# Database - SQLite by default (use PostgreSQL URL for production)
+DATABASE_URL="file:./dev.db"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)